kube-runtime/src/controller.rs

```diff
@@ -92,6 +99,75 @@ where
     })
 }
 
+/// A policy for when to retry reconciliation, after an error has occurred.
+pub trait ErrorPolicy {
+    type Err;
+    type K: RuntimeResource;
+    type Ctx;
+
+    /// Notifies that the state for an object should be removed, for example if it is reconciled successfully.
```

Comment:
Are we talking about the state of K in the reflector, or the state of the back off struct?